js/dist/wp-seo-admin-global-320.js

```diff
@@ -108,6 +108,117 @@
 
 	var $ = jQuery;
```

Comment:
That's an option, but the other method is used inside the file already so I decided to keep it uniform.